app/helpers/loss_report_helper.rb

```diff
@@ -0,0 +1,211 @@
+module LossReportHelper
+
+  def cmr_client
+    Cmr::Client.client_for_environment(Rails.configuration.cmr_env, Rails.configuration.services)
+  end
+
+  def token
+    if session[:login_method] == 'launchpad'
+      session[:launchpad_cookie]
+    elsif session[:login_method] == 'urs'
+      session[:access_token]
+    end
+  end
+
```

Comment:
the `cmr_client` and `token` methods are only here to allow the loss_report_helper_spec.rb to pass because without these two defined here, the spec file cannot access the definitions of `cmr_client` and `token` elsewhere. Keep in mind, the method definitions (`cmr_client` and `token`) aren't needed when being run in the app (as opposed to being run as a spec).